### ISSUE TITLE
[Port Request] I want Use in Firefox

### ISSUE THREAD

**Issue by Milor123** [mykeydoesotwork/dragon-bookmark-manager#1]
Hi dear !!! I want know, how can I use it in firefox?
I am in archlinux, i would like get it in firefox, I love this extension, its very fast and useful.

thank you very much !!

**Comment by ghost** [comment]
Agreed!!! I discovered this extension on Brave, and I love it! But I don't love chromium-based browsers. 
@mykeydoesotwork I can't find anything like this on Firefox. Or any other bookmark manager like this at all, period. It's awesome and deserves more recognition.
